stubs/google/cloud/tasks_v2/types.pyi

```diff
@@ -0,0 +1,6 @@
+from typing import Any, Text
+
+class Task:
+    name: Text = ...
+
+target_pb2: Any = ...
```

Comment:
Can we get rid of this `Any`?

Reply:
Here `target_pb2` is a protobuf message so that's the best we can do I think right??

Reply:
Hmm, I think that you might be able to import and set some type of it. Google repositories usually have generate protobuf stuff.

Reply:
Done as discussed offline.